feat: Introduce babel-*-udemy-* packages to the repo.

cansin opened the pull request:
##### *To:*
@udemy/team-f 

##### *What:*
Introduce babel-\*-udemy-\* packages to the repo. Later we’ll rename this repo from `eslint-udemy` to `public-npm-packages`.

##### *Trello:*
Part of https://trello.com/c/BYtm2RA4/849-move-our-babel-preset-udemy-eslint-plugin-a-and-eslint-config-udemy-into-udemy-org-on-github

##### *What did you test:*
I ran `yarn test` and verified tests for `babel-*` packages are running as well.

##### *What dashboards will you be monitoring:*
None for now.

Commit:
feat: Introduce babel-*-udemy-* packages to the repo.

Later we’ll rename this repo from `eslint-udemy` to `public-npm-packages`.

cansin commented on packages/babel-polyfill-udemy/.gitignore:
```diff
@@ -0,0 +1 @@
+helpers.js
```
@udemy/team-f basically, any change you see under `packages/babel-*` is the identical content of what https://github.com/udemy/babel-udemy repo currently have.

cansin replied:
I did not change any of the original source code/test content Dennis introduced.

cansin commented on .eslintignore:
```diff
@@ -0,0 +1,3 @@
+packages/babel-polyfill-udemy/helpers.js
```
I renamed `babel-environment-udemy`, and `babel-preset-udemy` as `babel-polyfill-udemy` and `babel-preset-udemy-website` respectively. But I am still debating which option would be the best:

1. `babel-polyfill-udemy` and `babel-preset-udemy-website`
1. `babel-polyfill-udemy-basics` and `babel-preset-udemy-basics`
1. `babel-polyfill-udemy-website` and `babel-preset-udemy-website`

@udemy/team-f opinions?

cansin commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Move our babel-preset-udemy, eslint-plugin-a, and eslint-config-udemy into Udemy org on GitHub](https://trello.com/c/BYtm2RA4/849-move-our-babel-preset-udemy-eslint-plugin-a-and-eslint-config-udemy-into-udemy-org-on-github)

kevinzang commented on packages/babel-preset-udemy-website/README.md:
```diff
@@ -0,0 +1,31 @@
+
+Install the Udemy Babel Preset locally.
+
+    $ yarn add babel-preset-udemy
```
Should this be babel-preset-udemy-website?

kevinzang reviewed (approved):
@cansin LGTM!

Commit:
test: Make tests more verbose.

Commit:
Publish

 - babel-polyfill-udemy-website@0.6.0
 - babel-preset-udemy-website@0.6.0
 - eslint-config-tester@0.1.1
 - eslint-config-udemy-babel-addons@1.0.8
 - eslint-config-udemy-basics@3.1.15
 - eslint-config-udemy-jasmine-addons@3.1.9
 - eslint-config-udemy-react-addons@3.1.9
 - eslint-config-udemy-website@4.0.3
 - eslint-plugin-udemy@4.0.2

cansin commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Move our babel-preset-udemy, eslint-plugin-a, and eslint-config-udemy into Udemy org on GitHub](https://trello.com/c/BYtm2RA4/849-move-our-babel-preset-udemy-eslint-plugin-a-and-eslint-config-udemy-into-udemy-org-on-github)

jjinux commented:
I thought there was a way to import a Git repo and its entire history into a subdirectory.
